src/main/kotlin/org/malachite/estella/organization/api/OrganizationController.kt

```diff
@@ -41,21 +41,19 @@ class OrganizationController(
 
 
     @CrossOrigin
-    @PostMapping("/addorganization")
+    @PostMapping()
     fun addOrganization(@RequestBody organization: OrganizationRequest): ResponseEntity<Organization> =
         organizationService.addOrganization(organization.toOrganization())
-            .let {
-                ResponseEntity.created(URI("/api/organizations/" + it.id)).body(it)
-            }
+            .let { ResponseEntity(it,HttpStatus.CREATED) }
 
     @CrossOrigin
     @PutMapping("/{organizationId}")
     fun updateOrganization(
         @RequestHeader(EStellaHeaders.jwtToken) jwt: String?,
         @PathVariable("organizationId") organizationId: OrganizationID,
         @RequestBody organization: OrganizationRequest
-    ): ResponseEntity<Message> {
-        if (!checkOrganizationUserRights(organizationId,jwt)) return OwnResponses.UNAUTH
+    ): ResponseEntity<Any> {
+        if (!checkOrganizationUserRights(organizationId, jwt)) return OwnResponses.UNAUTH
```

Comment:
My to w końcu we wtorek dogadaliśmy, czy chcemy to robić w kontrolerach, czy serwisach, czy gdzie? Bo z tego, co pamiętam, to gadaliśmy o tym, ale nie zdecydowaliśmy chyba w końcu

Reply:
Znaczy powiedzieliśmy chyba że będziemy to refaktorować przy dodawaniu autoryzacji na backu i wydaje mi się że wybraliśmy by robić to w serwisach odpowiednich